### PR TITLE
DOCS: Clarify steps in the contributing guidelines

### PR DESCRIPTION
**Pull request by joelostblom**
Just noticed a few things in the contributing guideline as I was making my recent PRs.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,52 +20,73 @@ slack](https://bit.ly/join-vega-slack).
...
 ```
-$ git checkout -b <branch-name>
+$ git switch -c <branch-name>
 ```
 With this branch checked-out, make the desired changes to the package.
+
 Note that Altair code uses the [black](https://black.readthedocs.io/)
-code formatter, which you can apply to your modifications by installing
-and running black on the local directory:
+and [flake8](https://flake8.pycqa.org/en/latest/) code formatters
```
Comment (jakevdp) [inline CONTRIBUTING.md:59]: nitpick: flake8 is a linter, not a code formatter

**Review by jakevdp — APPROVED** [review]
Looks great, thanks for putting this together! Just one minor nitpick

File: CONTRIBUTING.md (hunk 1 above)
Comment (joelostblom) [reply CONTRIBUTING.md]: Thank you, updated!

**Comment by joelostblom** [comment]
Actions are failing because of https://github.com/altair-viz/altair/pull/2528, not related to this PR

**Comment by jakevdp** [comment]
#2575 is merged, can you rebase this one?

**Comment by joelostblom** [comment]
Rebased and ready to merge!

**Comment by jakevdp** [comment]
Thanks!

**Pull request merged**